lib/workers/schedule_worker/schedules_builder.rb

```diff
@@ -0,0 +1,41 @@
+class ScheduleWorker < WorkerBase
+  class SchedulesBuilder
+    include Vmdb::Logging
+
+    attr_reader :schedules
+
+    def initialize(scheduler, queue, schedule_category = nil)
+      @scheduler = scheduler
+      @queue = queue
```

Comment:
:+1: i like this, and still manages to keep the diff mostly :red_circle: 

decided to go for a block instead of `Proc`, if that's OK?